mopidy/audio/tags.py

```diff
@@ -47,13 +47,13 @@ def convert_taglist(taglist):
                 try:
                     date = datetime.date(
                         value.get_year(), value.get_month(), value.get_day())
-                    result[tag].append(date.isoformat().decode('utf-8'))
+                    result[tag].append(date.isoformat())
```

Comment:
Just so I understand: this used to be decoded to unicode but now it's left as a native `str`? And the PY2 consumers of this need to be careful... except of course:

a) What happens under PY2 is now very uninteresting
b) It's only ever going to contain ASCII.

Reply:
I don't remember too well from a few weeks ago, but that sounds about right.